src/wallet/api.rs

```diff
@@ -1175,14 +1186,12 @@ impl Wallet {
         feerate: f64,
         manually_selected_outpoints: Option<Vec<OutPoint>>,
     ) -> Result<Vec<(ListUnspentResultEntry, UTXOSpendInfo)>, WalletError> {
-        log::info!(
-            "Coinselect Target : {} Sats with No. of Manually Selected Utxos: {}",
-            amount.to_sat(),
-            manually_selected_outpoints
-                .as_deref()
-                .unwrap_or_default()
-                .len()
-        );
+        if manually_selected_outpoints.is_some() {
+            log::info!(
+                "Coin selection target: {} sats with {manually_selected_outpoints:?} manually selected UTXOs",
+                amount.to_sat(),
+            );
+        }
```

Comment:
This log will bloat up due to Option<Vec> struct.  Also this will only be visible in manual selection case. I think this can be removed foir now. If we find a need for it later can be added back with better formatted values.